plugin/pkg/scheduler/factory/factory.go

```diff
@@ -115,57 +117,122 @@ func NewConfigFactory(client *client.Client, schedulerName string) *ConfigFactor
 		&api.Pod{},
 		0,
 		framework.ResourceEventHandlerFuncs{
-			AddFunc: func(obj interface{}) {
-				pod, ok := obj.(*api.Pod)
-				if !ok {
-					glog.Errorf("cannot convert to *api.Pod")
-					return
-				}
-				if err := schedulerCache.AddPod(pod); err != nil {
-					glog.Errorf("scheduler cache AddPod failed: %v", err)
-				}
-			},
-			UpdateFunc: func(oldObj, newObj interface{}) {
-				oldPod, ok := oldObj.(*api.Pod)
-				if !ok {
-					glog.Errorf("cannot convert to *api.Pod")
-					return
-				}
-				newPod, ok := newObj.(*api.Pod)
-				if !ok {
-					glog.Errorf("cannot convert to *api.Pod")
-					return
-				}
-				if err := schedulerCache.UpdatePod(oldPod, newPod); err != nil {
-					glog.Errorf("scheduler cache UpdatePod failed: %v", err)
-				}
-			},
-			DeleteFunc: func(obj interface{}) {
-				var pod *api.Pod
-				switch t := obj.(type) {
-				case *api.Pod:
-					pod = t
-				case cache.DeletedFinalStateUnknown:
-					var ok bool
-					pod, ok = t.Obj.(*api.Pod)
-					if !ok {
-						glog.Errorf("cannot convert to *api.Pod")
-						return
-					}
-				default:
-					glog.Errorf("cannot convert to *api.Pod")
-					return
-				}
-				if err := schedulerCache.RemovePod(pod); err != nil {
-					glog.Errorf("scheduler cache RemovePod failed: %v", err)
-				}
-			},
+			AddFunc:    c.informerAddPod,
+			UpdateFunc: c.informerUpdatePod,
+			DeleteFunc: c.informerDeletePod,
+		},
+	)
+
+	c.NodeLister.Store, c.nodePopulator = framework.NewInformer(
+		c.createNodeLW(),
+		&api.Node{},
+		0,
+		framework.ResourceEventHandlerFuncs{
+			AddFunc:    c.informerAddNode,
+			UpdateFunc: c.informerUpdateNode,
+			DeleteFunc: c.informerDeleteNode,
 		},
 	)
 
 	return c
 }
 
+func (c *ConfigFactory) informerAddPod(obj interface{}) {
```

Comment:
I feel this function name should be addPotToCache instead of informerAddPod. It is more clear about what this method really does. The informer is a consumer of this function, and when calling this function from the consumer side the consumer prefix seems to be duplicated too.